exercises/concept/log-levels/.docs/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+A `String` in Haxe is an object that represents immutable text as a sequence of Unicode characters (letters, digits, punctuation, etc.). Double quotes are used to define a `String` instance:
+
+```haxe
+var fruit:String = "Apple";
+```
+
+Strings are manipulated by calling the string's methods. Once a string has been constructed, its value can never change. Any methods that appear to modify a string will actually return a new string.
+
+Multiple strings can be concatenated (added) together. The simplest way to achieve this is by using the `+` operator.
+
+```haxe
+var name:String = "Jane";
+trace("Hello " + name + "!");
+// => "Hello Jane!"
+```
+
+For any string formatting more complex than simple concatenation, string interpolation is preferred. To use interpolation in a string, enclose it with single-quote characters (`'`). Then you can use the dollar sign (`$`) to trigger interpolation:
```

Comment:
> Not sure if string-interpolation should be a prerequisite concept here since it's used in the exemplar but also covered in introduction and hints. I followed what the C# track is doing for now.

So what the C# track did was to introduce basic string interpolation in the introduction (which you've copied) and to have a separate exercise going into more detail on string interpolation. We did that because string interpolation is such a commonly used concept in its basic form that we want to use it in many of the early exercises. It does deserve its own exercise though, hence our solution to this being the brief introduction.

Reply:
If you feel like string interpolation will be less important for other exercises, feel free to omit it here. That could actually make a lot of sense too, but it depends on what you want :)

Reply:
Ah, that makes sense. I was actually thinking that requiring students to go through the `string-interpolation` concept in order to unlock `strings` feels a little weird. Mentioning it in the brief here and having a separate concept/exercise pair that goes into the details seems like a good approach. Thanks for the input